src/main/java/org/broadinstitute/hellbender/tools/exome/CnvCaller.java

```diff
@@ -27,6 +28,8 @@
  *
  * Note:  Assumes that the incoming target file is in log_2(CR)
  * Note:  Assumes that the incoming segment means are CR.
+ * Note: Assumes that the input @{link ReadCountCollection} contains only one sample -- calls will be made for only one
```

Comment:
using one column of a matrix to represent a real vector quantity seems like a code smell to me.  the fact that "count" is in the name of the class (implying integer quantities) is also telling.  i think that having the appropriate abstractions is worth an extra class or two.